### PR TITLE
issue 789: show send button when draft message exists

### LINKED ISSUE #789: Show 'Send' button when draft message exists
Steps to reproduce: 
1. Open Room View screen
2. Type in 'New Message' input: 'draft message'
3. Click 'back' in header
4. Return back to the same Room View screen 

Expected: Draft message is displayed and the button on the right is 'Send'
Actual: Draft message is displayed but the button on the right is 'Microphone', which makes it not possible to send the draft right away after opening Room View screen.

Video: https://drive.google.com/file/d/1ubWcdFQo-L6z0r2wXb-NgbkqzZ2qvo6u/view?usp=sharing

- Your Rocket.Chat Experimental app version:  branch develop 1.11.0.100
- Your Rocket.Chat server version: open.rocket.chat
- Device (or Simulator) you're running with: simulator iOS iPhone X

### PR DESCRIPTION
**Pull request by IlarionHalushka**
@RocketChat/ReactNative
Closes #789 

Video: https://drive.google.com/file/d/1VNSHsPJ6cYnJlAT82MeJlelPJHcqhkd1/view?usp=sharing

### CODE REVIEW

File: app/containers/MessageBox/index.js [hunk 1]
```diff
@@ -111,8 +111,9 @@ export default class MessageBox extends Component {
 	componentDidMount() {
 		const { rid } = this.props;
 		const [room] = database.objects('subscriptions').filtered('rid = $0', rid);
-		if (room.draftMessage && room.draftMessage !== '') {
```
Comment (pranavpandey1998official) [inline app/containers/MessageBox/index.js]: don't remove this

File: app/containers/MessageBox/index.js [hunk 2]
```diff
@@ -111,8 +111,9 @@ export default class MessageBox extends Component {
...
+		if (room.draftMessage) {
 			this.setInput(room.draftMessage);
+			this.setState({ showSend: true });
```
Comment (pranavpandey1998official) [inline app/containers/MessageBox/index.js]: use function setShowSend

Comment (IlarionHalushka) [reply app/containers/MessageBox/index.js]: makes sense

File: app/containers/MessageBox/index.js (hunk 1 above)
Comment (IlarionHalushka) [reply app/containers/MessageBox/index.js]: why? room.draftMessage !== '' is redundant here

Comment (pranavpandey1998official) [reply app/containers/MessageBox/index.js]: K.. forgot that empty strings are also false in JS

Comment (IlarionHalushka) [reply app/containers/MessageBox/index.js]: no problem, thanks for taking a look at PR

**Comment by diegolmello** [comment]
Thanks for opening, but it's outdated already.

**Pull request closed without merging**